app/src/test/java/org/fossasia/openevent/app/presenter/EventDetailPresenterTest.java

```diff
@@ -192,4 +190,22 @@ public void shouldNotAccessView() {
 
         verifyZeroInteractions(eventDetailView);
     }
+
+    @Test
+    public void shouldHideProgressbarCorrectly() {
+        when(eventRepository.getAttendees(id, false))
+            .thenReturn(Observable.fromIterable(attendees));
+
+        when(eventRepository.getEvent(id, false))
+            .thenReturn(Observable.just(event));
+
+        InOrder inOrder = Mockito.inOrder(eventDetailView);
+
+        eventDetailPresenter.start();
+
+        assertTrue(eventDetailPresenter.getProgress() >= 2);
+
+        inOrder.verify(eventDetailView, atLeast(2)).showProgressBar(true);
```

Comment:
At least once please. 2 is a side effect of our implementation. If it is improved, we'll only be showing it once.